src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig

```diff
@@ -110,7 +110,7 @@
                                 <div class="input-field col s12">
                                     <h5>{{ 'config.form_settings.android_configuration'|trans }}</h5>
                                     <a href="wallabag://{{ app.user.username }}@{{ wallabag_url }}" class="waves-effect waves-light btn hide-on-large-only">{{ 'config.form_settings.android_instruction' | trans }}</a>
-                                    <img id="androidQrcode" class="hide-on-med-and-down" />
+                                    <img id="androidQrcode" class="hide-on-med-and-down" alt="QR code" />
```

Comment:
In order to translate this string, could you add a new translation key in this file: https://github.com/wallabag/wallabag/blob/master/src/Wallabag/CoreBundle/Resources/translations/messages.en.yml#L207 please?